models/edu/stanford/nlp/models/sutime/english.sutime.txt

```diff
@@ -888,37 +898,52 @@
 	           ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
     result: $0[-1].temporal.value
   }
-
   { name: "temporal-composite-7a",
     pattern: ( /every/ ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
-	result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
+	  result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
   }
-
   { name: "temporal-composite-7b",
-#    pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
+#   pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
```

Comment:
Is this still needed?

Reply:
This pattern is commented out upstream as well. I've removed it